packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -689,4 +791,27 @@ class _TableRowState<T> extends State<TableRow<T>>
 
   @override
   bool shouldShow() => widget.isShown;
+
+  void _handleSortChange(ColumnData<T> columnData) {
+    SortDirection direction;
+    if (columnData.title == widget.sortColumn.title) {
+      direction = widget.sortDirection.opposite();
+    } else if (columnData.numeric) {
+      direction = SortDirection.descending;
+    } else {
+      direction = SortDirection.ascending;
+    }
+    widget.onSortChanged(columnData, direction);
+  }
+}
+
+mixin TableSortMixin<T> {
```

Comment:
this is a bit of an odd mixin as the only mixed in functionality is private.
Consider making sortData part of an interface and moving the other two helpers as just private top level methods.

Reply:
done